src/lib/airtable/schema.js

```diff
@@ -29,7 +29,7 @@ export const Columns = {
 	"Sites": {
 		name: {name:`Name`, type:`text`},
 		userIds: {name:`Users`, type:`foreignKey-many`},
-		customerIds: {name:`Customers`, type:`foreignKey-many`},
+		customers: {name:`Customers`, type:`custom-object`}, // Modified via patch.
 		financialSummarieIds: {name:`Financial Summaries`, type:`foreignKey-many`},
 		tariffPlanIds: {name:`Tariff Plans`, type:`foreignKey-many`},
 		field6: {name:`Field 6`, type:`text`},
```

Comment:
looks like an unnecessary extra field was added into the Airtable and included in the generation

Reply:
Good catch. I'll remove it in another PR that'll also add a patch for the non-generated functions